cmake/mesh_dfu_metadata.cmake

```diff
@@ -22,4 +22,16 @@ function(mesh_dfu_metadata)
         ALL
         DEPENDS ${PROJECT_BINARY_DIR}/dfu_application.zip_ble_mesh_metadata.json
     )
+
+    add_custom_target(
+        # Prints already generated metadata
+        ble-mesh-dfu-metadata
```

Comment:
From what I see, names for custom commands use underscore instead of dash (except docs).